Mathlib/Data/Matrix/Rank.lean

```diff
@@ -119,6 +119,17 @@ lemma rank_mul_eq_right_of_isUnit_det [Fintype m] [DecidableEq m]
   have hAB : mulVecLin (A * B) = (LinearEquiv.ofIsUnitDet hA).comp (mulVecLin B) := by ext; simp
   rw [rank, rank, hAB, LinearMap.range_comp, LinearEquiv.finrank_map_eq]
 
+
```

Comment:
Only one line between declarations, per the [style guide](https://leanprover-community.github.io/contribute/style.html)
```suggestion
```